TrainingJava/src/House/ClientHouseBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package House;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author andreslietti
+ */
+
+// In this class the house is created requesting data from a client. 
+public class ClientHouseBuilder implements HouseBuilder {
+    private House house;
+    
+    
+    public ClientHouseBuilder(){
+        house = new House();
+    }
+    
+    @Override
+    public void buildStructure() {
+        System.out.println("Choose a structure: ");
+        String choice;
```

Comment:
If you need to get a default choice, this need to be assigned to null, otherwise, make the choice in one line like:
`String choice = scan.nextLine();`